### PR TITLE
docs: add vue3-flicking to frameworks banner

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This adds vue3-flicking banner to docs page

image

### CODE REVIEW

File: docs/src/pages/main/Frameworks.tsx [hunk 1]
```diff
@@ -18,6 +20,10 @@ export default () => {
       <div className="framework-logo-wrapper mr-2"><img src="img/icons/vue.svg" /></div>
       <span>@egjs/vue-flicking</span>
     </div>
+    <div className={`framework-logo button mr-2 ${styles["is-vue3"]}`}>
```
Comment (mixed) [inline docs/src/pages/main/Frameworks.tsx:23]: image
image

I think that You have to add these two areas to the README for more people to see. :)

**Review by mixed — APPROVED** [review]
LGTM

File: docs/src/pages/main/Frameworks.tsx (hunk 1 above)
Comment (WoodNeck) [reply docs/src/pages/main/Frameworks.tsx:23]: I'd like to, but isn't placing the same Vue icon twice looks bad on that page?

**Review by daybrush — APPROVED** [review]
LGTM

File: docs/src/pages/main/Frameworks.tsx (hunk 1 above)
Comment (mixed) [reply docs/src/pages/main/Frameworks.tsx:23]: Um... I agree enough too. I think it's okay to just put a number like this, but what do you think?

vue2
vue3

Comment (WoodNeck) [reply docs/src/pages/main/Frameworks.tsx:23]: I think that's too much.
We should leave more space for later frameworks like Svelte or so

Comment (mixed) [reply docs/src/pages/main/Frameworks.tsx:23]: OK. Can you tell me what the appropriate guideline are for you?

Comment (WoodNeck) [reply docs/src/pages/main/Frameworks.tsx:23]: I mean, that one Vue logo can both mean Vue2 & Vue3

We can rather show a message like "if you're using Vu3, check vue3-flicking" in the vue-flicking README

Comment (mixed) [reply docs/src/pages/main/Frameworks.tsx:23]: That's a good way. You can link vue2 readme and vue3 readme. If you see the trend later and there are more vue3s, you can change the main link to vue3 and I think it's a good idea.

Comment (WoodNeck) [reply docs/src/pages/main/Frameworks.tsx:23]: Good, I've updated the README

**Pull request merged**